game_api.txt

```diff
@@ -445,13 +445,13 @@ The player API can register player models and update the player's appearance.
 * `player_api.set_model(player, model_name)`
 	* Change a player's model
 	* `player`: PlayerRef
-	* `model_name`: model registered with player_api.register_model()
+	* `model_name`: model registered with `player_api.register_model`
```

Comment:
It is convention to include the argument brackets when referring to a function.

Reply:
`f()` refers to a call to the function without any arguments. I might decide to use `f(...)`, but I sure won't use `f()`.

Reply:
Ok, this is fine as it is, f(...) seems worse to me.